src/Diagnostics.RuntimeHost/Program.cs

```diff
@@ -24,15 +24,23 @@ public static IWebHost BuildWebHost(string[] args)
                         .AddCommandLine(args)
                         .Build();
 
-                    if (context.HostingEnvironment.IsDevelopment())
-                    {
+                    
                         var azureServiceTokenProvider = new AzureServiceTokenProvider();
                         var keyVaultClient = new KeyVaultClient(
                             new KeyVaultClient.AuthenticationCallback(
                                 azureServiceTokenProvider.KeyVaultTokenCallback));
 
+                    if (context.HostingEnvironment.IsDevelopment())
```

Comment:
Same here, can avoid copying code by just picking DevKeyVaultName / ProdKeyVaultName
May be worth making a static utility function to make the `$"https://{KeyvaultName}.vault.azure.net/"`